test/fixtures/block.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Fixtures.Block do
+  @moduledoc """
+  Fixtures for blocks.
+  """
+
+  alias Fixtures.Random
+
+  @spec beacon_block :: SszTypes.BeaconBlock.t()
+  def beacon_block do
+    %SszTypes.BeaconBlock{
+      parent_root: Random.root(),
+      slot: 1,
+      proposer_index: 1,
+      state_root: Random.root(),
+      body: beacon_block_body()
+    }
+  end
+
+  @spec beacon_block_body :: SszTypes.BeaconBlockBody.t()
+  def beacon_block_body do
+    %SszTypes.BeaconBlockBody{
+      randao_reveal: Random.bls_signature(),
+      eth1_data: eth1_data(),
+      graffiti: Random.hash(),
```

Comment:
Since _hash_ is not the specific type, we should call it `Random.bytes32()` or `Random.bytes(32)` to avoid confusion.